client/js/keybinds.js

```diff
@@ -70,6 +70,21 @@ Mousetrap.bind(["alt+shift+up", "alt+shift+down"], function(e, keys) {
 	return false;
 });
 
+Mousetrap.bind(["alt+a"], function() {
+	const sidebar = $("#sidebar");
+	const channels = sidebar
```

Comment:
Would be great if you could switch this filtering to use Vue state. We want to move away from jquery fully, and keybinds file has not been touched yet, but this looks like a great place to start.

Here's example code on how to loop through channels: https://github.com/thelounge/thelounge/blob/b4eb538903c7f2b1b865302829165591c7812cf9/client/js/utils.js#L58

Reply:
Thanks for the suggestion. I've re-written it using Vue, see the latest force-push.

I couldn't work out how to switch to the channel using Vue, so that is still using jquery in the same way that `client/js/commands/join.js` does it.

I've never touched Vue before today, so feel free to tell me more things that could be improved!

Reply:
> so that is still using jquery in the same way that client/js/commands/join.js does it.

Yeah channel/window switching hasn't been ported just yet.